### PR TITLE
Salesordergrid/coding standard

### PR DESCRIPTION
**Pull request by priyankavasoya**
@harishmk1805 
Please check this Module

### CODE REVIEW

File: app/code/Dss/SalesOrderGrid/etc/module.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ * that is bundled with this package in the file LICENSE.txt.
+ *
+ * @category   Dss
+ * @package    Dss_SalesOrderGrid
+ * @author     Extension Team
+ * @copyright Copyright (c) 2024 Digit Software Solutions. ( https://digitsoftsol.com )
+ */ -->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+    <module name="Dss_SalesOrderGrid" setup_version="1.0.0">
```
Comment (harishmk1805) [inline app/code/Dss/SalesOrderGrid/etc/module.xml:17]: remove setup_version

File: app/code/Dss/SalesOrderGrid/view/adminhtml/templates/sales/order/js.phtml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ *
+ * @category   Dss
+ * @package    Dss_SalesOrderGrid
+ * @author     Extension Team
+ * @copyright Copyright (c) 2024 Digit Software Solutions. ( https://digitsoftsol.com )
+ */ -->
+<?php
+/** @var $block \Dss\SalesOrderGrid\Block\Adminhtml\Sales\Order\Js */
+?>
+<script>
```
Comment (harishmk1805) [inline app/code/Dss/SalesOrderGrid/view/adminhtml/templates/sales/order/js.phtml:17]: use as per coding standards for magneto script tag

**Review by harishmk1805 — CHANGES_REQUESTED** [review]
please check changes

File: app/code/Dss/SalesOrderGrid/etc/module.xml (hunk 1 above)
Comment (priyankavasoya) [reply app/code/Dss/SalesOrderGrid/etc/module.xml]: @harishmk1805  changes uploaded please check

File: app/code/Dss/SalesOrderGrid/view/adminhtml/templates/sales/order/js.phtml (hunk 2 above)
Comment (priyankavasoya) [reply app/code/Dss/SalesOrderGrid/view/adminhtml/templates/sales/order/js.phtml]: @harishmk1805 
changes uploaded please check

**Review by harishmk1805 — APPROVED** [review]

**Pull request merged**